README.md

```diff
@@ -1,33 +1,80 @@
+  - We trust engineers to build the right things, collaborating closely and transparently with other teams.
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
 
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We have been one of the [first company in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
+    - We have been one of the first company [to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
+    - We have been one of the [first legacy company to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
+    - We have been one of the pioneer of [the concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
+ - We maintain our technical advantage through continuous innovation
+ - We aim to be at the forefront of tech, staying curious, looking to innovate and continuously improve.
 
+**Knowledge sharing**
 
+We believe that it’s our responsibility to upskill one another, sharing our knowledge, learnings and mistakes. 
 
+_Behaviours_:
+ - Individuals are entitled to pairing time for their development. 
+ - We make time for individual-learning & career development with personal development being our priority.
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [its values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```

Comment:
```suggestion
We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```

Makes the values more inclusive and matches the link location